monai/bundle/scripts.py

```diff
@@ -142,6 +144,35 @@ def _download_from_github(repo: str, download_path: Path, filename: str, progres
     extractall(filepath=filepath, output_dir=download_path, has_base=True)
 
 
+def _download_from_ngc(download_path: Path, filename: str, version: str, model_prefix: str, progress: bool):
+    url = _get_ngc_url(model_name=filename, version=version, model_prefix=model_prefix)
+    filepath = download_path / f"{filename}_v{version}.zip"
+    extract_path = download_path / f"{filename}"
+    download_url(url=url, filepath=filepath, hash_val=None, progress=progress)
+    extractall(filepath=filepath, output_dir=extract_path, has_base=True)
```

Comment:
Hi @yiheng-wang-nv @wyli ,

I think we should remove the "monai_" prefix in the extracted folder name to be consistent with model-zoo bundle.
What do you think?

Thanks.

Reply:
As discussed offline, please add the arg `remove_prefix: Optional[str] = "monai_"`. Will remove the prefix no matter where the bundle is downloaded, if can't find the prefix, just keep the same.
If set `remove_prefix=None`, will not remove anything.

Thanks.